### PR TITLE
locations and agreements fields

### PR DESCRIPTION
**Pull request by pronvit**

### CODE REVIEW

**Comment by pronvit** [comment]
a field in agreement refers to agreement_conclusion_reason enum, and 41-44 are petition reasons eradicate monsters/entertain/soldier/study but value 41 is taken in that enum for something else. probably there's a mistake in enum def. and maybe the enum should be renamed because it's not only conclusion reasons

File: df.entities.xml [hunk 1]
```diff
@@ -1167,9 +1167,9 @@
         <int32_t name='id'/>
         <stl-vector name='parties'>
             <pointer>
-                <int32_t/>
-                <stl-vector type-name='int32_t' ref-target='historical_figure'/>
-                <stl-vector type-name='int32_t'/>
+                <int32_t name='idx'/>
```
Comment (lethosor) [inline df.entities.xml]: Is this an index or an ID?

File: df.world-site.xml [hunk 2]
```diff
@@ -158,16 +158,16 @@
     <class-type type-name='abstract_building_inn_tavernst' inherits-from='abstract_building'>
         <compound name='name' type-name='language_name'/>
         <compound name='contents' type-name='abstract_building_contents'/>
-        <stl-vector>
+        <stl-vector name='room_info'>
             <pointer>
-                <int32_t/>
+                <int32_t name='idx'/>
```
Comment (lethosor) [inline df.world-site.xml]: (This too)

**Comment by pronvit** [comment]
I wasn't sure what name is better, changed to `id` now as I see `id` is being used in other situations like this.

**Comment by lethosor** [comment]
Yeah, we usually use "idx" as an abbreviation for "index", and that didn't seem to be an index into anything.

**Comment by pronvit** [comment]
Sure, having next_* fields imply they're not simple indices, I shouldn't have named them like that.

**Comment by pronvit** [comment]
What do you think about `agreement_conclusion_reason` enum?

**Comment by lethosor** [comment]
What about it?

**Comment by pronvit** [comment]
In my first message.

1. Agreement `reason` field has this type and I was going to append new values 41-44 for petition reasons to it. But value 41 already has [another label](https://github.com/BenLubar/df-structures/blob/eecb3ae1fb7b7099735c4d8c907f09a6f6359526/df.history.xml#L1449) in the enum. Most likely there's a mistake in the enum def, but don't know how to find where.

2. `agreement_conclusion_reason` doesn't seem to be the best name since these are not only 'conclusion' reasons.

**Pull request merged**